.github/workflows/test.yml

```diff
@@ -0,0 +1,54 @@
+name: Build and Test
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  build:
+    uses: liquibase/build-logic/.github/workflows/os-extension-test.yml@v0.5.2
+    secrets: inherit
+
+#  integration-test:
```

Comment:
@jandroav do we want to run integration tests?  Wouldn't we do this just like neo4j where the build/unit tests are reusable and then the integration tests are a separate step?

Reply:
Yes. Same as maxdb. Tests were commented on in the original workflow (ci.yml). We can uncomment them and see what happens. Who should take a look if tests have failures?

Reply:
@jandroav ah ok.  Lets handle that in separate ticket.  @filipelautert @kevin-atx who is the maintainer of this extension that could re-enable those tests?

Reply:
yes because no tests are executed:

```
      matrix:
       teradata: [ "" ]
```